azurerm/resource_arm_api_management.go

```diff
@@ -850,21 +886,49 @@ func flattenAzureRmApiManagementMachineIdentity(identity *apimanagement.ServiceI
 }
 
 func expandAzureRmApiManagementSku(d *schema.ResourceData) *apimanagement.ServiceSkuProperties {
```

Comment:
Can we split this into

`expandAzureRmApiManagementSku()` and `expandAzureRmApiManagementSkuName()` the the logic to pick in the calling function?

Reply:
Done.